Fixtures/Miscellaneous/ModuleAliasing/Simple/appPkg/Package.swift

```diff
@@ -2,23 +2,23 @@
 import PackageDescription
```

Comment:
I think you need to update the path on disk (in git) to also be AppPkg and  GamePkg, they look to be still camel case which would cause issues on case sensitive file systems. git can unfortunately be a pain about that

Reply:
strange, I renamed the paths but somehow the change is not showing up, will redo.